apps/src/code-studio/components/header/DisplayProjectName.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import ProjectUpdatedAt from './ProjectUpdatedAt';
+import styles from './project-header.module.scss';
+import classNames from 'classnames';
+
+export default class DisplayProjectName extends React.Component {
```

Comment:
Maybe a brief comment about this component, for newcomers?

Reply:
Same comment for other components too :)

Reply:
done!